example/src/MapViewExample.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { MapView, MapMarker, MapCallout, withTheme } from "@draftbit/ui";
+import { Dimensions, StyleSheet, Text, View } from "react-native";
+
+const MapViewExample = ({ theme }) => (
+  <View style={styles.container}>
+    <MapView
+      showsCompass
+      style={styles.map}
+      apiKey="AIzaSyA1rDXeD7tDxVMfP2bfwVXDEk8GEZ-kCAY"
```

Comment:
How sensitive is this? This is a public repo (cc @bluerssen)

Reply:
As discussed, I am okay to keep it there.